.github/workflows/permutation.yaml

```diff
@@ -0,0 +1,50 @@
+name: Permutation
```

Comment:
"Permutation" is quite a wide term (https://en.wikipedia.org/wiki/Permutation). Consider using naming that closely reflects the intent of this workflow (if I understand it correctly): "Upgrade testing"

Reply:
@somaritane , I'm not against. @k0da , @ytsarev  are you ok with "Upgrade testing", or "Bump testing" ?

Reply:
"Upgrade testing" is fine

Reply:
amended: Upgrade testing